pkg/minikube/bootstrapper/bsutil/kubeadm.go

```diff
@@ -150,6 +151,10 @@ func GenerateKubeadmYAML(cc config.ClusterConfig, n config.Node, r cruntime.Mana
 	if version.GTE(semver.MustParse("1.17.0")) {
 		configTmpl = ktmpl.V1Beta2
 	}
+	// handle the renaming of the coredns image from "coredns" to "coredns/coredns"
+	if version.GTE(semver.MustParse("1.21.0-alpha.1")) && opts.ImageRepository != "" {
```

Comment:
sorry for the late review on this PR,

do u mind clarifying is this change related to this PR ?
or could it be a separate PR ? to me it appears like a different PR's problem

Reply:
removed from current PR, split into #12411